File: C#/BluffinMuffin.Protocol/Lobby/JoinTableCommand.cs
using BluffinMuffin.Protocol.DataTypes.Attributes;
using BluffinMuffin.Protocol.Enums;

namespace BluffinMuffin.Protocol.Lobby
{
    /// <summary>
    /// To join a table, you have to send a **Join Table ** command with the id of the table you want to join.
    /// The success of the operation will be sent back
    /// </summary>
    public class JoinTableCommand : AbstractLobbyCommand, ICommandWithResponse<JoinTableResponse>
    {
        /// <summary>
        /// The id of the table to join
        /// </summary>
        [ExampleValue(42)]
        public int TableId { get; set; }

        /// <summary>
        /// 
        /// </summary>
        /// <returns></returns>
        public JoinTableResponse Response(bool success, BluffinMessageId msgId, string message)
        {
            return new JoinTableResponse(this) { Success = success, MessageId = msgId, Message = message };
        }
    }
}

File: Python/BluffinMuffin.Protocol/lobby/join_table_command.py
from abstract_lobby_command import AbstractLobbyCommand
from data_types.table_params import TableParams

__author__ = 'ericmas001@gmail.com'



class JoinTableCommand(AbstractLobbyCommand):
    def __init__(self, obj):
        super().__init__(obj)
        self.table_id = obj['TableId']

    def __str__( self ):
        return '{0} ({1})'.format(super().__str__(), self.table_id)
